examples/pennylane_run_state_preparation.py

```diff
@@ -101,7 +101,7 @@ def layer(params, j):
 # When defining the qnode, we introduce as input a Hermitian operator
```

Comment:
line `dev=qml.device` is producing the following warning inside the rendered documentation on [readthedocs](https://pennylane.readthedocs.io/en/doc-refactor/tutorials/pennylane_run_state_preparation.html)

```
/home/docs/checkouts/readthedocs.org/user_builds/pennylane/envs/doc-refactor/lib/python3.6/site-packages/pyquil/api/_compiler.py:300: UserWarning: No quilc server running at tcp://127.0.0.1:5555. Compilation using quilc will not be available.
```

Reply:
Hmm I'm not sure we can remove this, apart from having a Python line in the tutorial to suppress warnings. I can have this tutorial not autorun?

Reply:
Added a warnings filter to `doc/conf.py` :+1: